### PR TITLE
P.a.discussion

### PR DESCRIPTION
**Pull request by ksuess**
updated translation for https://github.com/plone/plone.app.discussion/pull/165

### CODE REVIEW

File: plone/app/locales/locales/de/LC_MESSAGES/plone.po [hunk 1]
```diff
@@ -6504,13 +6508,13 @@ msgstr "Es gab einen Fehler beim Speichern der Regeln."
 
 #: plone/app/theming/browser/controlpanel.py:220
 msgid "There were errors"
-msgstr "Fehler sind aufgetreten."
+msgstr "Fehler sind augetreten."
```
Comment (jensens) [inline plone/app/locales/locales/de/LC_MESSAGES/plone.po]: typo?

File: plone/app/locales/locales/de/LC_MESSAGES/plone.po [hunk 2]
```diff
@@ -6504,13 +6508,13 @@ msgstr "Es gab einen Fehler beim Speichern der Regeln."
...
 
 #: plone.app.caching/plone/app/caching/browser/controlpanel.py:310
 #: plone.app.users/plone/app/users/browser/membersearch.py:94
 #: plone.app.users/plone/app/users/browser/register.py:67
 msgid "There were errors."
-msgstr "Fehler sind aufgetreten."
+msgstr "Fehler sind augetreten."
```
Comment (jensens) [inline plone/app/locales/locales/de/LC_MESSAGES/plone.po]: typo?

**Comment by mauritsvanrees** [comment]
Possible problem is that the master branch of `plone.app.locales` is used on both coredev 5.1 and 5.2. So when we merge this, the translations in 5.1 will be less good. I have not checked what the difference would be.

I guess this could be true for other packages as well, where we have different branches in 5.1 and 5.2. No big problem, but something to keep in mind.

**Comment by mauritsvanrees** [comment]
@vincentfretin I have just merged the related `plone.app.discussion` PR into Plone 5.2. This `plone.app.locales` PR contains the locales changes. Small problem is what I said in my [previous comment above](#issuecomment-596476728): on 5.1 some good translations will be gone.

Do you want to do anything about that? Maybe copy some existing msgids to a `manual.pot` so the existing translations do not get lost?

I guess if you do that (or someone else), then this PR can be removed, except that a few German translations were added.

**Comment by ksuess** [comment]
I'll add the lost 5.1 translation to a discussion-manual.pot

**Comment by vincentfretin** [comment]
@ksuess You can add all removed msgid with the default message in plone-manual.pot, there is already a "Plone 5.1 messages removed in Plone 5.2" section in there. You should go back to plone.app.locales master, run `bin/i18n plone` in a buildout.coredev 5.2 and add all removed messages in plone-manual.pot. Commit that file.
Double check you really added all deleted messages, revert all other changes `git checkout .` in src/plone.app.locales and run `bin/i18n plone` again. You should have 0 deleted message.

**Comment by ksuess** [comment]
I added the translations (update of plone-manual.pot and readded translations in po-files) that are needed for Plone 5.1 which will not use the new plone.app.discussion release.

Nevertheless, there are warnings on other packages translations.
```
/Users/ksuess/Plone/coredev52multipy/py2/eggs/zope.tal-4.4-py2.7.egg/zope/tal/talgettext.py:155: UserWarning: Warning: msgid 'label_existingTagsHelp' in Products.Archetypes/Products/Archetypes/skins/archetypes/widgets/tests/keywordtests.html:(78, 36) already exists with a different default (bad: Use Control/Command/Shift keys to select multiple tags. ${DYNAMIC_CONTENT}, should be: Use Control/Command/Shift keys to select multiple tags.)
The references for the existent value are:
Products.Archetypes/Products/Archetypes/skins/archetypes/widgets/keyword.pt:(48, 33)

  references.encode('utf-8')))
Warning: msgid 'Please correct the indicated errors.' in Products.CMFPlone/Products/CMFPlone/browser/login/password_reset.py:196 already exists with a different default (bad: , should be: None)
The references for the existent value are:
Products.Archetypes/Products/Archetypes/skins/archetypes/validate_base.vpy:19
Products.Archetypes/Products/Archetypes/skins/archetypes/validate_integrity.cpy:20
Warning: msgid 'This field is required, please provide some information.' in Products.CMFPlone/Products/CMFPlone/skins/plone_form_scripts/validate_content_status_modify.vpy:15 already exists with a different default (bad: None, should be: )
The references for the existent value are:
Products.CMFPlone/Products/CMFPlone/browser/login/password_reset.py:191
Warning: msgid 'An integer is required.' in plone.app.caching/plone/app/caching/browser/controlpanel.py:298 already exists with a different default (bad: , should be: None)
The references for the existent value are:
plone.app.caching/plone/app/caching/browser/controlpanel.py:288
plone.app.caching/plone/app/caching/browser/controlpanel.py:279
Warning: msgid 'label_description' in plone.app.dexterity/plone/app/dexterity/behaviors/metadata.py:77 already exists with a different default (bad: Summary, should be: Description)
The references for the existent value are:
Products.Archetypes/Products/Archetypes/ExtensibleMetadata.py:90
Warning: msgid 'help_effective_date' in plone.app.dexterity/plone/app/dexterity/behaviors/metadata.py:151 already exists with a different default (bad: If this date is in the future, the content will not show up in listings and searches until this date., should be: The date when the item will be published. If no date is selected the item will be published immediately.)
The references for the existent value are:
plone.app.content/plone/app/content/browser/content_status_history.py:21
Products.Archetypes/Products/Archetypes/ExtensibleMetadata.py:137
Warning: msgid 'help_expiration_date' in plone.app.dexterity/plone/app/dexterity/behaviors/metadata.py:161 already exists with a different default (bad: When this date is reached, the content will no longer be visible in listings and searches., should be: The date when the item expires. This will automatically make the item invisible for others at the given date. If no date is chosen, it will never expire.)
The references for the existent value are:
plone.app.content/plone/app/content/browser/content_status_history.py:31
Products.Archetypes/Products/Archetypes/ExtensibleMetadata.py:148
```

**Comment by vincentfretin** [comment]
Those are the usual warnings, you can ignore.

**Comment by vincentfretin** [comment]
There is something wrong with some default messages in your last commit. Anyway, the commit where  you add the removed messages in plone-manual.pot should be the first, then a resync and update translations. So I did https://github.com/collective/plone.app.locales/commit/dbe6a8f3c4dc89de47447bc101ab0d01575ea3cf and https://github.com/collective/plone.app.locales/commit/56aa946672520e33b5fed0890139a20b559ef9f1
You can now update your translation again on master. You can remove this branch.

**Comment by vincentfretin** [comment]
FYI, I reverted one of my previous resync https://github.com/collective/plone.app.locales/commit/43e3a4ba77aeb2b620097009b9aa810713bc4d76 that was included in the plone.app.locales 5.1.21 release that broke compatibility with Plone 5.1.x. I modified the mail_notification_message_moderator msgid https://github.com/plone/plone.app.discussion/commit/181fee785b1aa6a63b3d55bb99b74cad1d449be8 and resynced again to keep old translation for Plone 5.1.x. This was related to your previous https://github.com/plone/plone.app.discussion/pull/163 PR @ksuess

**Pull request closed without merging**